### ISSUE TITLE
Background color/image configuration

### ISSUE THREAD

**Issue by BowlBird** [coastalwhite/lemurs#157]
Hello, I know that this is pretty lightweight and I'm unsure of the full capabilities of TUIs in general, however I think it would be cool if Lemurs could support setting an image or color for the background. Thanks :)

**Comment by coastalwhite** [comment]
Yeah, I think this is quite a reasonable idea and should not be too difficult with `tui-rs`. I am not sure how it would look in an actual TTY though.

**Comment by danma3x** [comment]
Shouldn't this be pretty straightforward with what we have in `ratatui`? 

So, for background color, we could simply render a fullscreen `Block` first.
With an image, however, it's a bit of a different story, there's a lot of ways one could go about it and knobs to add in the config.rs. 

With rendering itself, `Canvas` or a custom widget seems like it would be enough. But it would add a dependency on image decoding libraries, which is concerning, even if we hide it behind a feature flag.

**Comment by coastalwhite** [comment]
Yeah, it should be fairly easy to add a background colour.

I would be in favour of adding this as a background colour first. Then, maybe we can do a similar thing to Ly and add some different background effects.
